packages/editor/src/components/zoom-out-toggle/index.js

```diff
@@ -37,6 +37,8 @@ const ZoomOutToggle = () => {
 
 	return (
 		<Button
+			accessibleWhenDisabled
+			disabled={ disabled }
```

Comment:
TIL about this new prop.

Interesting that we don't just set `accessibleWhenDisabled` to `true` by default. I assume there's some logical reason why that's not the case.

Reply:
Yeah, I only learned about it thanks to the linter! As far as I can tell it's optional in Ariakit (https://ariakit.org/reference/button#accessiblewhendisabled), so I imagine our Button component has it optional either for consistency or back compat. I agree it feels a bit redundant, though, having to set it in a bunch of places as in Gutenberg I imagine we'd want to be using it all the time.

Good to keep in mind for future consolidation, for sure 👍